src/ucs/config/parser.c

```diff
@@ -760,7 +760,7 @@ int ucs_config_sscanf_array(const char *buf, void *dest, const void *arg)
     int ret;
     unsigned i;
 
-    str_dup = strdup(buf);
+    str_dup = ucs_strdup(buf, "config_scanf_array str_dup");
```

Comment:
it is inconsistent with other changes made in this PR. You did not add `str_dup` to the name of other `ucs_strdup` calls

Reply:
not sure I got your message... str_dup is variable name

Reply:
i mean that the name passed to `ucs_strdup` contains `str_dup`. I'd use `config_scanf_array`

Reply:
ok, renamed